server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1921,18 +1835,34 @@ public void deleteConsumer(
         sink.queueEvent(event);
     }
 
-    @ApiOperation(notes = "Retrieves a list of Entitlement Certificates for the Consumer",
-        value = "getEntitlementCertificates")
-    @ApiResponses({ @ApiResponse(code = 404, message = "") })
-    @GET
-    @Path("{consumer_uuid}/certificates")
-    @Produces(MediaType.APPLICATION_JSON)
-    @UpdateConsumerCheckIn
-    public List<CertificateDTO> getEntitlementCertificates(
-        @PathParam("consumer_uuid") @Verify(Consumer.class) String consumerUuid,
-        @QueryParam("serials") String serials) {
-
+    /**
+     * Method to get entitlement certificates.
+     * NOTE: Here we explicitly update consumer Check-In.
+     *
+     * @param consumerUuid
+     *  Consumer UUID
+     *
+     * @param serials
+     *  Certificate serial
+     *
+     * @return
+     *  List of DTOs representing certificates
+     */
+    public List<CertificateDTO> getEntitlementCertificates(@Verify(Consumer.class) String consumerUuid,
+        String serials) {
         log.debug("Getting client certificates for consumer: {}", consumerUuid);
+
+        // UpdateConsumerCheckIn
```

Comment:
Correct. Since this method is called indirectly, ```ConsumerCheckInFilter``` filter is not applied. Therefore we are explicitly updating consumer check-in.